app/containers/Applications/ApplicationListContainer.tsx

```diff
@@ -29,14 +29,14 @@ export const ApplicationList = props => {
     'Facility Name': 'facility_name',
     'Reporting Year': 'reporting_year',
     'Submission Date': 'submission_date',
-    Status: 'application_status'
+    Status: 'application_revision_status'
   };
   const body = (
     <tbody>
       {edges.map(edge => (
         <ApplicationRowItemContainer
-          key={edge.node.id}
-          ciipApplication={edge.node}
+          key={edge.node.rowId}
```

Comment:
The search result function does not return a relay id. I think it's because the id is not a primary key as it can change with each result. It's created with row_number() over () in the sql query. So relay only creates a rowId for us.